oauth/loginmanager.py

```diff
@@ -1,10 +1,12 @@
 from oauth.services.github import GithubOAuthService
 from oauth.services.google import GoogleOAuthService
+from oauth.services.rhsso import RHSSOOAuthService
 from oauth.oidc import OIDCLoginService
 
 CUSTOM_LOGIN_SERVICES = {
     "GITHUB_LOGIN_CONFIG": GithubOAuthService,
     "GOOGLE_LOGIN_CONFIG": GoogleOAuthService,
+    "RHSSO_LOGIN_CONFIG": RHSSOOAuthService,
```

Comment:
I don't think oidc implements the same interface as these services (`login_enabled`).
You might need to add an extra check when instantiating the oidc service.